pkg/auto/exporthttp/job/job.go

```diff
@@ -99,22 +124,53 @@ func (b *BaseJob) GetUrl() string {
 	return b.Url
 }
 
-func (b *BaseJob) GetExecutionAfter(t time.Time) <-chan time.Time {
-	if t.IsZero() {
-		t = time.Now().UTC()
+func (b *BaseJob) GetAutoName() string {
+	return b.AutoName
+}
+
+func (b *BaseJob) GetScName() string {
+	return b.ScName
+}
+
+func (b *BaseJob) GetExecutionAfter(jobName string) <-chan time.Time {
+	t := time.Now().UTC()
+	// check the previous execution timestamp
+	previous := time.Time{}
+	key := fmt.Sprintf(boltKeyTemplate, b.GetAutoName(), b.GetScName(), jobName)
```

Comment:
If you use the field `b.AutoName` and `b.ScName` here you can remove the methods from the Job interface.